elyra/cli/pipeline_app.py

```diff
@@ -138,8 +150,8 @@ def _preprocess_pipeline(pipeline_path: str,
         raise click.ClickException(f"Error pre-processing pipeline: \n {e}")
 
     if not _validate_pipeline_runtime(primary_pipeline, runtime):
-        runtime_description = primary_pipeline['app_data']['properties']['runtime']
-        runtime_config_display_name = _get_runtime_display_name(runtime_config)
+        runtime_description = _get_runtime_display_name(primary_pipeline['app_data']['runtime'])
+        runtime_config_display_name = _get_runtime_display_name_from_config(runtime_config)
```

Comment:
I found this confusing when in this area recently.  I don't understand the naming or the methods.  Might I suggest the following?
```suggestion
        pipeline_runtime_name = _get_runtime_display_name(primary_pipeline['app_data']['runtime'])
        requested_runtime_name = _get_runtime_display_name(_get_runtime_type(runtime_config))
```
and remove `_get_runtime_display_name_from_config()`

Reply:
Much better indeed, thanks @kevin-bates